lua/swenv/api.lua

```diff
@@ -197,11 +200,16 @@ M.auto_venv = function()
     if not project_venv_name then
       return
     end
-    local closest_match = best_match(venvs, project_venv_name)
-    if not closest_match then
-      return
+    -- in-project venv activation
+    local venv = { path = get_local_venv_path(project_dir), name = project_venv_name }
+
+    -- venvs folder actication
+    if not venv.path then
```

Comment:
would this ever be false?

Reply:
Good catch, it won't. I am going to update this this afternoon.